woof-code/rootfs-skeleton/etc/rc.d/rc.network_eth

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
```

Comment:
@rserwin1 Can you reopen this PR after removing all the whitespace/indentation changes? This code is very messy, and it's hard to review it when most changes are cosmetic.